.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+AllCops:
+  Exclude:
+    - bin/**/*
+    - db/**/*
+    - tmp/**/*
+    - vendor/**/*
+    - log/**/*
+
+  RunRailsCops: true
+  DisplayCopNames: true
+  DisplayStyleGuide: true
+  StyleGuideCopsOnly: true
```

Comment:
These four settings make sense, they can help us to quickly understand a Cop :+1: